### PR TITLE
Adding initial PLIO support to the platform

### PR DESCRIPTION
**Pull request by eddierichter-amd**
This is linked to the [mlir-aie PR](https://github.com/Xilinx/mlir-aie/pull/1623) that adds PLIO support to the compiler and has an example that targets this platform.

### CODE REVIEW

**Comment by stephenneuendorffer** [comment]
Are you really still using 2022.1?  updating to 2023.2 would be good, which is where most of the other work is, although I expect we'll move to 2024.2 once it's released.

**Comment by eddierichter-amd** [comment]
> Are you really still using 2022.1? updating to 2023.2 would be good, which is where most of the other work is, although I expect we'll move to 2024.2 once it's released.

It is a good point, I made an issue to track this: https://github.com/Xilinx/ROCm-air-platforms/issues/14. Note that this version is solely for building the platform, so this does not preclude using 2023.2 or whatever newer version for mlir-aie (2023.2 is what I currently use), only that you need 2022.1 to build the platform. That being said, I get that now that we are opening up the platform for users to integrate their PL designs it makes sense to update the Vitis version for the platform as well.

File: platform/vck5000/aie_platform/Makefile [hunk 1]
```diff
@@ -56,6 +56,13 @@ AIE_CMPL_CMD = aiecompiler -platform=${PLATFORM} -include="./aie" -workdir=./Wor
 AIE_SIM_CMD = aiesimulator --pkg-dir=./Work --dump-vcd foo
 EMU_CMD = ./launch_hw_emu.sh
 
+# Putting where we put the PL source code
+PL_SRC_REPO  := ./pl
+COUNTER_KERNEL_TOP := counter_hls
+COUNTER_KERNEL_SRC := $(PL_SRC_REPO)/counter/$(COUNTER_KERNEL_TOP).cpp
+COUNTER_KERNEL_XO  := $(COUNTER_KERNEL_TOP).$(TARGET)
+COUNTER_KERNEL_VPP_FLAGS := --hls.clock $(VPP_CLOCK_FREQ):$(COUNTER_KERNEL_TOP)
+
```
Comment (muwyse-amd) [inline platform/vck5000/aie_platform/Makefile:65]: use "=" instead of ":="

**Review by muwyse-amd — APPROVED** [review]
LGTM with minor change

**Pull request merged**